storybook/src/stories_v7/textarea/textarea.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { TextInput, type TextInputProps } from '@equinor/fusion-react-textarea/src/TextArea';
+
+const meta: Meta<TextInputProps> = {
+  title: 'data/Textarea',
+  component: TextInput,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const basic: Story = {
+  args: {
+    label: 'basic',
+  },
+};
+
+export const size: Story = {
+  args: {
+    label: 'Multiple rows',
+    rows: 6,
+  },
+};
+
+export const required: Story = {
+  args: {
+    label: 'required',
+    required: true,
+  },
+};
+
+export const disabled: Story = {
+  args: {
+    label: 'required',
+    disabled: true,
+  },
+};
+
+// export const charCounter: Story = {
+//   args: {
+//     label: 'Character counter',
+//     charCounter: true,
+//   },
+// };
```

Comment:
Please uncomment or remove.

Reply:
nope, that story should work, so leave trace for future fix, no time atm